app/scripts/shell/dimAppCtrl.controller.js

```diff
@@ -246,64 +130,5 @@
         refresh();
       }
     }, false);
-
-    function addFilter(filter) {
-      var input = $('input[name=filter]');
-      var itemNameFilter = false;
-
-      if (filter === 'item name') {
-        itemNameFilter = true;
-        filter = $window.prompt("Enter an item name:");
-        filter = filter.trim();
-      }
-
-      if (filter.indexOf('light:') === 0 || filter.indexOf('quality:') === 0) {
```

Comment:
in the new version, we lost the `quality:` filter function

https://github.com/DestinyItemManager/DIM/pull/715/files#diff-42a67061c6db1d1b99fb8fa2c8637c6dR28